.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,114 @@
+      label: Role
+      description: For what role is the feature you want to propose meant to be?
+      options:
+        - All
+        - sap_anydb_install_oracle
+        - sap_general_preconfigure
+        - sap_ha_install_anydb_ibmdb2
+        - sap_ha_install_hana_hsr
+        - sap_ha_pacemaker_cluster
+        - sap_hana_install
+        - sap_hana_preconfigure
+        - sap_hostagent
+        - sap_install_media_detect
+        - sap_maintain_etc_hosts
+        - sap_netweaver_preconfigure
+        - sap_storage_setup
+        - sap_swpm
+        - None of them (or I don't know)
+    validations:
+      required: true
+
+  - type: dropdown
+    id: os-family
+    attributes:
+      label: OS
+      description: What operating system are you using to run the role?
+      options:
+        - RHEL (7.x, 8.x, 9.x)
+        - SLES (15 SPx)
+        - None of the above (specify in the description area below)
+      default: 2
+    validations:
+      required: true
+
+  - type: textarea
+    id: python-version
+    attributes:
+      label: Python
+      description: Please provide the python version you are using
+      placeholder: 3.9.7
+    validations:
+      required: true
+
+  - type: textarea
+    id: ansible-version
+    attributes:
+      label: Ansible-core
+      description: Please provide the ansible-core version you are using
+      placeholder: 2.12.0
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: Please provide a way for us to be able to reproduce the problem you ran into.
+      placeholder: Reproduction
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: |
+        A clear and concise description of what the bug is.
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+      placeholder: Bug description
+    validations:
+      required: true
+
+  - type: dropdown
+    id: additional
+    attributes:
+      label: Additional information
+      description: Do you intend to submit a pr to solve this bug?
+      options:
+        - "Yes"
+        - "No"
+      default: 1
+    validations:
+      required: true
+
+  - type: textarea
+    id: additonal
+    attributes:
+      label: Additional context
+      description: If applicable, add any other context about the problem here
+
+  - type: checkboxes
+    id: required-info
+    attributes:
+      label: Final checks
+      description: Before submitting, please make sure you do the following
+      options:
+        - label: I've read [the documentation](https://github.com/sap-linuxlab/community.sap_install/tree/main/docs#readme).
+          required: true
+        - label: I've already check for existing duplicated [issues](https://github.com/sap-linuxlab/community.sap_install/issues?q=is%3Aissue+label%3Abug).
```

Comment:
grammar: check -> checked

My proposals:
"I have already searched for existing duplicate issues and found none"
Alternative: 
"I have already searched for existing duplicate issues." plus:
- [checkbox] I found none.
- [checkbox] I found the following which might be related (add links to issues): [free text entry field]

Reply:
Why does someone need to open a new issue instead of commenting on the existing one?

Reply:
My point was that users should be urged to search for existing issues and then report if they found none or if they found possibly related issues. In the latter case, they may of course decide not to open a new issue. Or they believe that their issue is not completely covered by an existing issue.

Reply:
Make sense, I've updated the template with a text area, see https://github.com/sap-linuxlab/community.sap_install/pull/831/commits/59842f557f3b3571fd4005eaa3dd4592253879d4